### PR TITLE
Add device replacement use-case

### PR DESCRIPTION
**Pull request by tlwr**
OP https://mailarchive.ietf.org/arch/msg/green-bof/YMn-D_lMms-0bCju6UHzZlgft1c/

As per email thread with Emile, adding into draft as use-case

### CODE REVIEW

File: draft-stephan-legacy-path-eco-design.md [hunk 1]
```diff
@@ -137,6 +137,25 @@ The tracking of LCM information may reveal device usage information.
 
 Device inventory can be used by attackers when assessing a system for vulnerable components.
 
+# Use Cases
+
+## Device replacement
```
Comment (emile22) [inline draft-stephan-legacy-path-eco-design.md:142]: in a DC

**Review by emile22 — APPROVED** [review]
many thanks

**Comment by emile22** [comment]
great!

**Comment by tlwr** [comment]
Thank you!

**Pull request merged**